data/human/pirate jobs.txt

```diff
@@ -3139,26 +3108,48 @@ mission "Syndicate Assassination [1]"
 		government "Syndicate"
 		personality staying target
 		system
-			distance 2 5
+			government "Syndicate"
+			distance 2 3
 		fleet
 			names civilian
-			variant
-				"Star Queen"
+			variant 6
+				"Arrow"
+			variant 6
+				"Bounder"
+			variant 2
+				"Arrow"
 				"Quicksilver" 2
+			variant 2
+				"Bounder"
+				"Manta"
+			variant 3
+				"Arrow"
+				"Wasp" 3
+			variant 3
+				"Bounder"
+				"Quicksilver"
+				"Wasp" 2
+			variant 1
+				"Arrow"
+				"Splinter"
+			variant 1
+				"Bounder"
+				"Splinter"
 		dialog "The target has been destroyed. Return to <destination> for your payment."
 	on complete
-		payment 350000
+		payment 460000
 		dialog "A pirate warlord thanks you for eliminating the politician and pays you <payment>."
 		"reputation: Syndicate" -= 10
 		"reputation: Pirate" += 3
 		
 mission "Syndicate Assassination [2]"
 	name "Assassinate a political target"
-	description "A Syndicate politician who has plans that worry many local pirate gangs was spotted in their ship, the <npc>, in a nearby system. Assassinate them in order to send a message to the Syndicate. Payment is <payment>. Be advised that they are heavily defended."
+	description "A Syndicate politician who has plans that worry many local pirate gangs was spotted in their ship, the <npc>, in a nearby system. Assassinate them by <day> in order to send a message to the Syndicate. Payment is <payment>. Be advised that they may be heavily defended."
```

Comment:
This is looking pretty long. Shaving off a few words to try and avoid it spilling out of the UI.
```suggestion
	description "A Syndicate politician with plans that worry local pirate gangs was spotted in their ship, the <npc>, in a nearby system. Assassinate them by <day> to send a message to the Syndicate. Payment is <payment>. Be advised that they may be heavily defended."
```